### DISCUSSION TITLE
[1245620]ELDEN RING艾尔登法环 {本体+dlc}

### DISCUSSION THREAD

**Discussion by tymolu233** [pjy612/SteamManifestCache#703] [Share]
### Steam 游戏ID

1245620

### 密钥列表

```hash
1245621;5f560fca048a138487e1e634a1e60693a7ad3ae7318c31731d147f16e093f10b
1245624;e194ce5e5f83c427743781c1708c4bce6683d5d65521059f6e4ca64a8f36f3cd
1896300;946e5c4e2c91a3ed363efc688218994f7fa6e8fdd56aa8b5752456d3b28e0543
2778580;9f1556645ea8ef43529f920cf02a2682a6da5756b29e630ba376a0cde24e3908
2855520;476eca9191866d6743aa7ad82d4d7ce1fcd5e0f0613f2f4b6559635d68f8c0e3
```

### 清单附件

[1245621_8868449847359611551.zip](https://github.com/user-attachments/files/15936065/1245621_8868449847359611551.zip)

### 备注

内测码: CFB1D118AB9E8E4D
邮箱: tymolu@outlook.com

**Comment by ghost** [comment]
下载完没有eldenring.exe这个文件

**Reply by ghost** [reply]
手动下载这个文件就可以进了

**Comment by ZEERDEER** [comment]
补充一点密钥：
ELDEN RING Shadow of the Erdtree - Artbook & Soundtrack
[2855520_164164058551746673.zip](https://github.com/user-attachments/files/15935022/2855520_164164058551746673.zip)

`2855520` 
```
476eca9191866d6743aa7ad82d4d7ce1fcd5e0f0613f2f4b6559635d68f8c0e3
```

**Reply by LMP42** [reply]
还差一个1245623的密钥

**Reply by ZEERDEER** [reply]
> 还差一个1245623的密钥

那个没用
1245623 ELDEN RING JP
1245624 ELDEN RING ROW
互相冲突 ROW = Rest of the World

**Reply by LMP42** [reply]
> > 还差一个1245623的密钥
> 
> 那个没用 1245623 ELDEN RING JP 1245624 ELDEN RING ROW 互相冲突 ROW = Rest of the World

但是软件提示我就是差这个密钥，所以解锁不了

**Reply by pjy612** [reply]
实际上 不影响

**Comment by xokmanx** [comment]
你好！请问你是否拥有《Elden Ring》的1.11版本或对应的manifest ID？我一直在寻找这个版本，希望能亲自测试和分析它的内容。非常感谢你的分享，如果你能提供相关信息，我将不胜感激！

**Reply by pjy612** [reply]
貌似就没有过 1.11。
1.09 直接到 1.12 了 自己去Steamdb 查

**Reply by xokmanx** [reply]
https://eldenring.fandom.com/wiki/Game_Version/1.11

**Reply by pjy612** [reply]
> https://eldenring.fandom.com/wiki/Game_Version/1.11

https://steamdb.info/app/1245620/patchnotes/
那你对着 Patch 记录 一点一点扣 ManifetId 然后 针对性的下载吧

https://steamdb.info/patchnotes/11683484/
https://steamdb.info/patchnotes/13018193/

这俩里面应该有一个是的

**Reply by xokmanx** [reply]
在 SteamDB 上目前找不到 Elden Ring 的 1.11 版本。如果您能协助我定位该版本，我将非常感激。这个版本对我的研究非常重要，尤其是用于比对 Bullet 参数的结构变化。期待您的帮助，谢谢！

**Reply by pjy612** [reply]
我已经回复过了

https://steamdb.info/patchnotes/11683484/
https://steamdb.info/patchnotes/13018193/

这俩里面应该有一个是的

**Reply by xokmanx** [reply]
目前在 GitHub 或 SteamDB 上找不到 Elden Ring 的 1.11 版本。这个版本似乎只提供给媒体或评测人员，用于抢先体验 DLC。如果您能协助我获取该版本，我将非常感激。谢谢您的帮助！
